### ISSUE TITLE
Double clicking an item on the list will not keep its selection

### ISSUE THREAD

**Issue by ZakDanger** [joxeankoret/diaphora#109]
Once a compare between 2 databases has been done it presents multiple lists such as "Best match", "Partial match" etc. If you select an item in one of these lists and press enter or double click it to jump to the assembly for that item's function, that list will not remember what item it was that you were on when you go back to it. In other words it loses its selection or highlighted item.

If you are going through a list checking out various items this can be annoying because you can't easily keep track of where you are up to.

It turns out this issue is due to the code that is part of IDA. I have reported it and it will be fixed for the next release of IDA, but if you want to fix it in your current version of IDA to fix this issue then do the following.

## TL;DR
Patch ```IDA/python/ida_kernwin.py``` line 6236 from:

    idx = n.pop(0)
to

    idx = n[0]

**Comment by joxeankoret** [comment]
Thanks a lot for reporting it here! I hope the Hex-Rays guys fix it shortly. I will leave it open for some time so others with the same problem can find easily this issue and its solution.

**Comment by joxeankoret** [comment]
It's fixed with IDA 7.1. Closing.
